front/src/components/Chat/JoinLeave.tsx

```diff
@@ -15,7 +15,7 @@ export function CheckPassword(props: { trigger: boolean, setTrigger: (value: boo
 	const [, setInputValue] = useState<string>("");
 
 	const handleJoinWithPass = () => {
-		socket.emit('joinChannel', { chanid: props.channel.id, channame: props.channel.name, password: password });
+		socket.emit('joinChannel', { chanid: props.channel.id, channame: props.channel.name, password: password }); // can we remove channame ? don't remember why we had it
```

Comment:
Right, I don't think we need chan name. I forgot why we had it too lol
I think we should remove chan name